JamulusServer.entitlements

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.cs.allow-dyld-environment-variables</key>
+	<true/>
+	<key>com.apple.security.cs.disable-library-validation</key>
```

Comment:
Didn't look this up what is disable-library-validation?

Reply:
> Almost ready, I think? Did anybody already add you as contributor in the app?

Yeah I'm in there already - thanks for checking.
I backed out the icon line - need to find the proper way for that to work.

I've made one more change, and made the entitlements as minimal as I can and enabled sandboxing. The entitlements are now the same for both client and server - an artifact of using UDP and needing to essentially open a server connection on both client and server.

Here is the sandboxed build for testing: https://upload.disroot.org/r/vN0aHdMu#I4xVeSGbETmXnJb6gZbbWr4LWcePoW/0Y3Ka6v+paqs=